app/pb_kits/playbook/pb_date_picker/_date_picker.scss

```diff
@@ -0,0 +1,1018 @@
+      // justify-content: flex-start;
+      padding-top: 0px;
+    }
+    .flatpickr-monthDropdown-months {
+      appearance: none;
+      padding-left: 0px;
+      padding-right: 14px;
+      margin-left: 15px;
+      height: $header_height;
+      &:hover {
+        background-color: white;
+      }
+    }
+    .numInputWrapper {
+      display: contents;
+      .numInput {
+        appearance: none;
+        border-left: solid 1px $border_light;
+        padding-left: 22.5px;
+        padding-right: 14px;
+        margin-left: 10px;
+        height: $header_height;
+        &:focus {
+          outline: none;
+        }
+        &:hover {
+          cursor: pointer;
+        }
+      }
+    }
+
+    .flatpickr-prev-month {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      left: 234px !important;
+      padding-right: 10px;
+      border-left: solid 1px $border_light;
+      height: $header_height;
+      // svg path {
+      //   fill: $neutral;
+      // }
+      &:hover{
+        svg {
+          fill: inherit;
+        }
+      }
+    }
+    .flatpickr-next-month {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding-left: 10px;
+      height: $header_height;
+      // svg path {
+      //   fill: $neutral;
+      // }
+      &:hover{
+        svg {
+          fill: inherit;
+        }
+      }
+    }
+
+    div.flatpickr-weekdays {
+      margin-top: 8px;
+    }
+    span.flatpickr-weekday {
+      @include caption;
+      user-select: none;
+    }
+    // Month-Year Select Styles
+    select.flatpickr-monthDropdown-months {
+      font-size: 16px;
+      font-weight: 600;
+      color: #242B42;
+      margin-top: 0px;
+      margin-bottom: 0px;
+    }
+
+    select.numInput {
+      font-size: 16px;
+      font-weight: 600;
+      color: #242B42;
+      border: 0px;
+      font-family: inherit;
+    }
+
+    // Calendar Shadow
+    .flatpickr-calendar {
+      box-shadow: $shadow_deeper;
+    }
+
+    // Day and Range Styles
+    .flatpickr-day {
+      user-select: none;
+      &:hover {
+        background-color: rgba($color: $neutral, $alpha: 0.2);
+        border-color: rgba($color: $neutral, $alpha: 0.2);
+        box-shadow: $shadow_deep;
+      }
+    }
+    
+    .flatpickr-day.today {
+      font-weight: 600;
+      color: $charcoal;
+      &.selected {
+        color: $text_dk_default;
+      }
+    }
+    .flatpickr-day.selected, .flatpickr-day.startRange, .flatpickr-day.endRange {
+      background-color: $primary;
+      border-color: $primary;
+      box-shadow: $shadow_deep;
+      font-weight: 600;
+    }
+    // had to use flat hex colors without alpha channels
+    // the shadows were overlapping and opacities were adding together in spots
+    .flatpickr-day.inRange {
+      background-color: #E5EEFA;
+      border: 2px;
+      box-shadow: -5px 0 0 #E5EEFA, 5px 0 0 #E5EEFA;
+    }
+  }
+
+  &.error {
+    [class^=pb_body_kit] {
```

Comment:
can we use a class instead?

Reply:
Changed to unique class name `.error-body-kit`